.gitignore

```diff
@@ -5,3 +5,4 @@
 .DS_Store
 venv/
 .pytest_cache/
+**/__pycache__/
```

Comment:
Do not make any additional changes in the `.gitignore` file, your PR should include only changes made in the `main.py`.